Src/Contract/WindowsCredential.cs

```diff
@@ -109,7 +109,8 @@ public override void ApplyTo(ChannelFactory channelFactory)
 
             X509ChainPolicy chainPolicy = new X509ChainPolicy();
             chainPolicy.VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority | X509VerificationFlags.IgnoreNotTimeValid;
-            channelFactory.Credentials.ServiceCertificate.Authentication.CustomCertificateValidator = X509CertificateValidator.CreateChainTrustValidator(true, chainPolicy);
+            // TODO - this is missing from .net standard, but it may not be possible to do Windows auth safely without it
+            //channelFactory.Credentials.ServiceCertificate.Authentication.CustomCertificateValidator = X509CertificateValidator.CreateChainTrustValidator(true, chainPolicy);
```

Comment:
This comment is concerning. We should follow up.

Reply:
See the changes in https://github.com/solarwinds/OrionSDK/pull/271/commits/b051e847866c207183f6d5c7d623a199f3588a30.  I'm hoping that will do the trick.